CHANGES/1026.bugfix.rst

```diff
@@ -0,0 +1,20 @@
+Joining URLs with empty segments has been changed
+to match :rfc:`3986`.
+
+Previously empty segments would be removed from path,
+breaking use-cases such as
+
+.. code-block:: python
+
+   URL("https://web.archive.org/web/") / "https://github.com/"
+
+Now :meth:`~yarl.URL.__truediv__` and :meth:`~yarl.URL.joinpath` operation
```

Comment:
How about something like this?
```suggestion
Now :meth:`/ <yarl.URL.__truediv__>` and :meth:`URL.joinpath() <yarl.URL.joinpath>` operation
```

@commonism would such rendering make sense to you?